app/controllers/api/verify/document_capture_errors_controller.rb

```diff
@@ -1,27 +0,0 @@
-module Api
-  module Verify
-    class DocumentCaptureErrorsController < BaseController
-      include EffectiveUser
-
-      def delete
-        form = DocumentCaptureErrorsDeleteForm.new(
```

Comment:
Looks like `DocumentCaptureErrorsDeleteForm` is unused now and could be removed.